### DISCUSSION TITLE
可以使用import方式引入么

### DISCUSSION THREAD

**Discussion by xcyeye** [leancloud/javascript-sdk#655] [Q&A] [answered]
我项目使用vite，但是按照官网那样引入，会出现require is not defined，我试过使用import引入，好像不行，想问问有没有什么办法解决

**Comment by leeyeh** [comment, marked as answer]
```
import { init, Query } from 'leancloud-storage'
```

**Reply by xcyeye** [reply]
> ```
> import { init, Query } from 'leancloud-storage'
> ```

好的，非常感谢
